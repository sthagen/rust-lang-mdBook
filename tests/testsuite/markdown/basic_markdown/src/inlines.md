# Inlines

*emphasis* **bold** **_bold emphasis_**

Some `inline code`.

Hard\
break

Invisible hard  
break

\[escaped] \<html> \*here*
