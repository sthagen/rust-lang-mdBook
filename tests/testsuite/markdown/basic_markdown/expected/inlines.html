<h1 id="inlines"><a class="header" href="#inlines">Inlines</a></h1>
<p><em>emphasis</em> <strong>bold</strong> <strong><em>bold emphasis</em></strong></p>
<p>Some <code>inline code</code>.</p>
<p>Hard<br>break</p>
<p>Invisible hard<br>break</p>
<p>[escaped] &lt;html&gt; *here*</p>